libs/blocks/aside/aside.css

```diff
@@ -140,11 +195,10 @@
 
 .aside.split {
   flex-direction: column;
-  background-color: #1E1E1E;
 }
 
-.aside.split.light {
-  background-color: white;
+.aside.split.dark {
+  background-color: #1E1E1E;
```

Comment:
Same comment as above. Might want to check w/ the team if these should be author-able via the first row decorator. Hard coded colors aren't very forgiving.

Reply:
We can actually override these but I think removing the defaults will have some issues backward compatibility